validation/README.md

```diff
@@ -0,0 +1,116 @@
+
+## Context
+
+1.  Data validation must be performed for/by a range of data custodians. Many different data custodians generate wastewater surveillance data, including academic testing labs, national and commercial testing labs, municipal wastewater utilities and public health. These data custodians require different types of support to validate the data they generate in ODM format.
+
+2.  Data validation must be performed on different platforms. Data custodians have many different data systems. Most academic testing labs use Excel. Many data custodians do not use open programming software such as R or Python. ArcGIS and Microsoft 365 are used by key stakeholders. An example of a platform-specific validation is missing data. The ODM format for missing data is to leave the value blank. However, different program platforms specific format for missing data to facilitate analyses. Missing data formats vary considerably between platforms. Missing data can also be coded in different formats within a programming language. The use of platform-specific missing data tokens allows users of the platform to use a consistent missing data format.
+
+3.  Data validation must be performed for ODM users with a range of analytic skills and knowledge. The extent of data validation for most users is limited, and there are considerable inconsistencies in how data is currently recorded. Many data custodians do not have formal data science training or capacity.
+
+4.  Data validation is required at different stages of the data life cycle, including: during generation of data within the lab and environments of data custodian; prior and during transmission from data custodians to data repositories; within data repositories; when data is transferred from one data repository to another; before data analyses by epidemiologists and modellers.
+
+## Validation components
+
+ODM supports validation using four components:
+
+**Validation rules**: Examples of validation rules include validating measure data types (e.g. integer, string), ranges (e.g. number between 0 and 100), uniqueness (e.g. sample ID is not repeated). The ODM supports a core set of validation rules for the ODM dictionary. 
+
+**Validation schema:** Validation rules are described using a validation schema. Users can use the ODM validation schema to generate additional rules. The validation schema can be used to validate data in other settings, such as other environmental dictionaries or databases.
+
+**Validation library:** The ODM supports a Python library to validate data. Developers can use the Python library in their applications to validate data. 
+
+**Validation application:** The ODM will have a publicly available web application that users can use to validate their data. The rules are also machine-actionable using an open format. 
```

Comment:
`The rules are also machine-actionable using an open format.`, maybe this should be part of the "Validation schema" component.

Reply:
Agreed